ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/cv-pathfinder-edge1.cfg

```diff
@@ -320,11 +320,16 @@ interface Dps1
    flow tracker hardware FLOW-TRACKER
    ip address 192.168.142.2/32
 !
-interface Ethernet1
-   description Inmrasat_S511
+interface Ethernet1/49
    no shutdown
    no switchport
+!
+interface Ethernet1/49.3
+   description Inmrasat_S511
+   no shutdown
+   encapsulation dot1q vlan 3
    ip address dhcp
+   ip access-group TEST-IPV4-ACL-WITH-IP-FIELDS-IN_Ethernet1/49.3 in
```

Comment:
I am not too happy about these names. Please replace the dot with underscore. The slash should also be replaced, but I think we need to wait for 5.0 with that part.